docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+De procesdefinitie is in veel gevallen zelfs impliciet (hard gecodeeerde flow tussen schermen); dat valt dan niet eens los te knippen.
+
+Conclusie: hier is sprake van een glijdende schaal. Je wilt liefst alle domeingegevens en zo veel mogelijk procesgegevens uit de applicaties halen en via een gestandaardiseerde APi ontsluiten. We moeten echter wel oppassen dat we hier te ver in willen gaan en het paard daarmee achter de wagen spannen. In de praktijk zal dit voor veel bestaande taaksystemen een groeipad worden..
+
+Er is één maar. Het delen van procesinformatie is nodig om het handelen als overheid te verantwoorden: het moet dus gearchiveerd worden. Wie heeft op welke datum welke stap gezet? Welke procesflow (en keuzes) is er doorlopen? Welke informatie was wanneer beschikbaar? Hoe luidde de (deel)berekening die heeft geleid tot een bepaald resultaat? Dit is informatie die niet in één van de andere registraties past (tenzij als logfile in PDF in de DRC) en voor het grootste deel niet in de ZRC wordt vastgelegd (die kent meestal maar één behandelaar en vooral enkel het proces op hoofdlijnen: de Statussen).
+Dit pleit voor een procesverloopregistratie, waarin op een zo gestandaardiseerde manier als mogelijk informatie over het verloop van alle processen wordt bijgehouden. Probleem is echter dat het ene proces het andere niet is en het erg moeilijk is dit gestandaardiseerd te doen (een zaak probeert dat feitelijk ook, en deze kent niet voor niets enkel statussen). Dus wellicht krijg je dan een subsiedieprocesverloopregistratie, een vergunningprocesverloopregistratie, een burgerzakenprocesverloopregistratie, … Dat is dan weer niet aanlokkelijk.
+Wellicht dat een combinatie met de desbetreffende domeinregistratie dan nog het meest voor de hand ligt. Een domeinregistratie Vergunningen (en handhaving), waarin –volgens een gestandaardiseerd informatiemodel- naast de in het vergunningproces betrokken objecttypen, ook het verloop van de vergunningprocessen wordt geregistreerd.
+
+### 1.4.3. Relatie proces – zaak
+
+In de figuur staat ook een stippellijn getekend tussen Zaak en Procesgegevens. Proces staat niet gedefinieerd in het informatiemodel RGBZ, en dus ligt de relatie zaak – proces ook niet eenduidig vast.
+In GFO zaken was dit min of meer wel het geval met “stap”. Hieruit, en vooral uit de prominente rol van ‘status’ in het RGBZ, blijkt al dat het moeilijk is om alle soorten processen op een eenduidige manier te registreren. Uiteindelijk kom je dan bij een informatiemodel met de rijkheid van BPMN uit (en voor de meer adaptieve processen iets dat lijkt op CMMN). Wat uiteraard niet de bedoeling is.
+Maar indachtig de vorige vraag: “waar registreren we procesinformatie”, is het noodzakelijk dat de relatie zaak-proces goed is gedefinieerd, zodat we als we nadere procesinformatie willen weten van een zaak, deze bij een zaaknummer kunnen opvragen (indien beschikbaar).
+
+### 1.4.4. Redundantie van gegevens in de ZRC?
```

Comment:
Zou dit niet gewoon ook moeten? Zie mijn vorige opmerking over het procesverloop?

Reply:
eigenlijk wel. maar het is wat ambitieus..zie boven